src/sagemaker_sklearn_container/training.py

```diff
@@ -28,9 +28,11 @@ def train(training_environment):
                                training arguments and hyperparameters
     """
     logger.info('Invoking user training script.')
-    framework.modules.run_module(training_environment.module_dir, training_environment.to_cmd_args(),
-                                 training_environment.to_env_vars(), training_environment.module_name)
+    framework.modules.download_and_install(training_environment.module_dir)
+    framework.entry_point.run(training_environment.module_dir, training_environment.user_entry_point,
+                              training_environment.to_cmd_args(), training_environment.to_env_vars(),
+                              runner=framework.runner.ProcessRunnerType)
```

Comment:
This is the default runner already right? Was this included for awareness?

Reply:
Yes